aiida/backends/tests/test_restapi.py

```diff
@@ -876,6 +876,76 @@ def test_structure_attributes_filter(self):
             response = json.loads(rv_obj.data)
             self.assertEqual(response['data']['nodes'][0]['attributes']['cell'], cell)
 
+    ############### node attributes_filter with pagination #############
+    def test_node_attributes_filter_pagination(self):
+        """
+        Check that node attributes specified in attributes_filter are
+        returned as a dictionary when pagination is set
+        """
+        expected_attributes = ['resources', 'cell']
+        url = self.get_url_prefix() + '/nodes/page/1?perpage=10&attributes=true&attributes_filter=resources,cell'
+        with self.app.test_client() as client:
+            response_value = client.get(url)
+            response = json.loads(response_value.data)
+            self.assertNotEqual(len(response['data']['nodes']), 0)
+            for node in response['data']['nodes']:
+                self.assertIn('attributes', node)
+                self.assertNotIn('attributes.resources', node)
+                self.assertNotIn('attributes.cell', node)
+                self.assertEqual(len(node['attributes']), len(expected_attributes))
+                for attr in expected_attributes:
+                    self.assertIn(attr, node['attributes'])
+
+    ############### node get one attributes_filter with pagination #############
+    def test_node_single_attributes_filter(self):
+        """
+        Check that when only one node extra is specified in extras_filter
```

Comment:
You still have "extra" twice in this docstring but should be "attribute".

Reply:
Sorry, fixed it.